bin/commands/channels.rb

```diff
@@ -38,6 +38,7 @@
   g.long_desc %( Gets information about a channel. )
   g.command 'info' do |c|
     c.flag 'channel', desc: 'Channel to get info on.'
+    c.flag 'include_locale', desc: 'Set this to true to receive the locale for this channel. Defaults to false.'
```

Comment:
This is new, so needs a changelog line too.

Reply:
Yep, it's hidden in the middle of that one long changelog item I wrote 😉 . I'll split it up into 3 items.